node/src/manager/commands/rewind.rs

```diff
@@ -104,30 +107,24 @@ pub async fn run(
     };
 
     println!("Pausing deployments");
-    let mut paused = false;
     for deployment in &deployments {
-        if let Some(node) = &deployment.node_id {
-            if !node.starts_with(PAUSED) {
-                let loc = deployment.locator();
-                let node =
-                    NodeId::new(format!("{}{}", PAUSED, node)).expect("paused_ node id is valid");
-                subgraph_store.reassign_subgraph(&loc, &node)?;
-                println!("  ... paused {}", loc);
-                paused = true;
-            }
-        }
-    }
+        let search = match searches.iter().find(|s| s.locate_unique(&primary).is_ok()) {
```

Comment:
The type `DeploymentSearch` can correspond to different deployments. `locate_unique` just pops one deployment that correspond to a search, we want all of them. 

So a cleaner way to do it would be to modify `pause_or_resume` to take a `DeploymentLocator` instead of `DeploymentSearch1 this way we could just only call `pause_or_resume(deployment)` inside this loop,

Reply:
Okay, I'lll revise.